src/course-outline/subsection-card/SubsectionCard.jsx

```diff
@@ -9,25 +9,29 @@ import { setCurrentItem, setCurrentSection, setCurrentSubsection } from '../data
 import { RequestStatus } from '../../data/constants';
 import CardHeader from '../card-header/CardHeader';
 import BaseTitleWithStatusBadge from '../card-header/BaseTitleWithStatusBadge';
+import ConditionalSortableElement from '../drag-helper/ConditionalSortableElement';
 import TitleButton from '../card-header/TitleButton';
-import { getItemStatus, scrollToElement } from '../utils';
+import { getItemStatus, getItemStatusBorder, scrollToElement } from '../utils';
 import messages from './messages';
 
 const SubsectionCard = ({
   section,
   subsection,
   children,
+  index,
+  canMoveItem,
```

Comment:
Should this be false when there are no other siblings, such as when there is only on subsection in a section? There is no where to move the element so it shouldn't have the drag handle or the move options in the menu.

Reply:
> Should this be false when there are no other siblings, such as when there is only on subsection in a section?

Yes, both move up & down options are disabled.

> There is no where to move the element so it shouldn't have the drag handle

Makes sense, updated :+1: